plugins/feeds/public/blocklist_de_apache_ip.py

```diff
@@ -0,0 +1,42 @@
+from datetime import timedelta
+import logging
+
+from core.observables import Ip
+from core.feed import Feed
+from core.errors import ObservableValidationError
+
+
+class BlocklistdeApacheIP(Feed):
```

Comment:
I would capitalize the `DE` in `Blocklistde` to be consistent with the other feed names. (same goes for the `default_values` dict below)

Reply:
Done.